src/Castle.Windsor.Tests/Compatibility/InterfaceMapping.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+#if FEATURE_NETCORE_REFLECTION_API
+namespace System.Reflection
+{
+	internal static class InterfaceMappingReflectionExtensions
+	{
+
+		public static InterfaceMapping GetInterfaceMap(this Type implType, Type interfaceType)
```

Comment:
Is there a reason we can't use `GetRuntimeInterfaceMap` on .NET Core like we did in Castle Core (https://github.com/castleproject/Core/blob/master/src/Castle.Core/Compatibility/RuntimeReflectionExtensions.cs)?